cfgov/v1/blocks.py

```diff
@@ -51,6 +51,7 @@ class HeadingLevelBlock(blocks.ChoiceBlock):
         ("h2", "H2"),
         ("h3", "H3"),
         ("h4", "H4"),
+        ("h5", "H5"),
```

Comment:
I think this change will require a new Django migration. You'll need to run something like

```
cfgov/manage.py makemigrations v1 --name heading_block_h5s
```

to generate the migration file.

Reply:
done!